eox_tenant/signals.py

```diff
@@ -223,6 +223,9 @@ def start_async_tenant(sender, *args, **kwargs):  # pylint: disable=unused-argum
     See:
        https://celery.readthedocs.io/en/latest/userguide/signals.html#task-prerun
     """
+    if not getattr(base_settings, "USE_EOX_TENANT", False):
```

Comment:
@morenol note here that we also need to turn this on for Async servers when going to prod.